recipes/aiocontextvars/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "aiocontextvars" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "1e0ff5837c8b01c36a1107acdd0baf7853ebdf6c9fc43e8e311f4be37ac2038a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
nit: I think the flavour of the week is to use `{{ PYTHON }}`

Reply:
Yes. This should be changed.